src/java.base/share/classes/module-info.java

```diff
@@ -425,6 +422,17 @@
     provides java.util.random.RandomGenerator with
         java.security.SecureRandom,
         java.util.Random,
-        java.util.SplittableRandom;
+        java.util.SplittableRandom,
+
```

Comment:
Can you remove this blank files, otherwise the list of service types for this provide declaration are all together?